python/paddle/fluid/trainer.py

```diff
@@ -124,7 +124,10 @@ def test(self, reader):
 
     def save_params(self, param_path):
         # reference: save_persistables in io.py
-        pass
+        exe = executor.Executor(self.place)
+        exe.run(self.startup_program, scope=self.scope)
```

Comment:
I am not sure if we need this. I guess `save_persistables` will call `save_vars` which will run the executor (https://github.com/PaddlePaddle/Paddle/blob/0ecc6fa8f35d9bc15d605db82551ead8f502d523/python/paddle/fluid/io.py#L129)

Reply:
Ah I see. So 
```
        save_program = Program()
        save_block = save_program.global_block()
```
already loads all the variables from the scope?

Reply:
Discussed offline.